Site/ASAB Maestro/Descriptors/kibana.yaml

```diff
@@ -96,9 +98,9 @@ nginx:
     - proxy_method POST
     - proxy_set_body ""
     - proxy_set_header X-Request-URI "request_uri"
-      # proxy_cache batman_responses;
+    - proxy_cache kibana
     - proxy_pass http://upstream-seacat-auth-public/batman/nginx?client_id=kibana
-      # ??? proxy_cache_key $cookie_SeaCatSCI_XXX;
-      # proxy_cache_lock on;
-      # proxy_cache_valid 200 10s;
+    - proxy_cache_key $cookie_SeaCatSCI_QBOAFDNWZ52WDP22
```

Comment:
There should be a comment about why this looks like it looks.

Reply:
I tried... I know like... 50% of how this works...